package.json

```diff
@@ -76,13 +78,15 @@
     "rss": "^1.2.2",
     "source-map-explorer": "^2.5.2",
     "tempy": "^0.6.0",
+    "terser-webpack-plugin": "^5.3.3",
     "tus-js-client": "^2.3.0",
     "twemoji": "^14.0.2",
     "videojs-contrib-ads": "^6.9.0",
     "videojs-ima": "^1.11.0",
     "videojs-ima-player": "^0.5.6",
     "videojs-logo": "^2.1.4",
-    "videojs-vtt-thumbnails": "https://github.com/OdyseeTeam/videojs-vtt-thumbnails"
+    "videojs-vtt-thumbnails": "https://github.com/OdyseeTeam/videojs-vtt-thumbnails",
+    "write-file-webpack-plugin": "^4.5.1"
```

Comment:
Should `write-file-webpack-plugin` be under `devDependencies` instead, or doesn't matter?

Reply:
Ah, yes. You're probably correct. It's been a while, but this was probably a mistake. You're right that if it is a dev dependency it doesn't matter from a functional perspective. But I can make this adjustment to keep from exacerbating any organization issues.